custom_components/easee/const.py

```diff
@@ -60,6 +61,17 @@
     },
 }
 OPTIONAL_EASEE_ENTITIES = {
+    "temp_max": {
+        "key": "state.tempMax",
+        "attrs": [],
+        "units": TEMP_CELSIUS,
+        "convert_units_func": "round_0_dec",
+        "device_class": SensorDeviceClass.TEMPERATURE,
+        "state_class": SensorStateClass.MEASUREMENT,
+        "icon": "mdi:temperature-celsius",
```

Comment:
```suggestion
```
This icon is wrong if user changes the unit of measurement. Remove and let HA use standard icon based on device_class.